force-app/main/default/lwc/placeholder/__tests__/placeholder.test.js

```diff
@@ -0,0 +1,36 @@
+import { createElement } from 'lwc';
+import Placeholder from 'c/placeholder';
+
+/*
+ * By default @salesforce/lwc-jest will resolve the
+ * @salesforce/resourceUrl/bike_assets import to be "bike_assets". We could
+ * validate that string is present in the test, but we'll override the default
+ * import for exemplary purposes.
+ */
```

Comment:
As the comment says, I overrode the default mock mostly for another example of dealing with the `@salesforce` module imports. I can remove and update the test though if we think we already have enough examples of that.

Reply:
I’d prefer to find a better suited example to show how to override an `@salesforce` import